docs/spec/relayer-v2/Chain.tla

```diff
@@ -0,0 +1,146 @@
+---------------------------- MODULE Chain ----------------------------
+
+EXTENDS Naturals, FiniteSets, RelayerDefinitions, 
+        ClientHandlers, ConnectionHandlers, ChannelHandlers
+        
+CONSTANTS ChainID, \* chain identifier
+          Heights \* set of possible heights of the chains in the system 
+
+VARIABLES chainStore,
+          incomingDatagrams
+
+vars == <<chainStore, incomingDatagrams>>          
+
+(***************************************************************************
+ Client update operators
+ ***************************************************************************)
+\* Update the clients on chain with chainID, 
+\* using the client datagrams generated by the relayer      
+\* (Handler operators defined in ClientHandlers.tla)
+LightClientUpdate(chainID, chain, datagrams) == 
+    \* create clients
+    LET clientCreatedChain == HandleCreateClient(chainID, chain, datagrams) IN
```

Comment:
I didn't know we can use operators from different files like this. Is there a way we can prefix it somehow so it is more clear where it comes from?

Reply:
This is because `Chain.tla` extends the module `ClientHandlers.tla`. Then the operators defined in `ClientHandlers.tla` are incorporated in `Chain.tla`, and are used as if they were defined in `Chain.tla`.

Reply:
I guess there is no way to somehow namespace differently imported operators: for example clientHandlers.HandleCreateClient as it is imported from the file ClientHandlers.tla?